harmonysoft-micrometer/src/main/kotlin/tech/harmonysoft/oss/micrometer/registry/StatsRegistry.kt

```diff
@@ -51,102 +51,27 @@ class StatsRegistry<T : Meter>(
         cacheEntryCreator(staticTags)
     }
 
-    fun getMeter(): T {
-        if (tagNames.isNotEmpty()) {
+    fun getMeter(vararg tagValues: String): T {
```

Comment:
the whole idea of having methods with different parameters is to avoid using vararg methods. The reason is that in runtime that results in creating an array to wrap given parameters. If stats code is called frequently (and it's supposed to be called frequently) that would be unnecessary memory usage.

I'm ok to add **new** method with varargs, but not by **replacing** existing methods by it